pages/common/zfgrep.md

```diff
@@ -0,0 +1,25 @@
+# zfgrep
+
+> Matches fixed strings in possibly compressed files.
+> Equivalent to `grep -F` but first decompresses input if necessary.
+> More information: <https://www.gnu.org/software/grep/manual/grep.html>.
+
+- Search for an exact string in a file:
+
+`zfgrep {{search_string}} {{path/to/file}}`
```

Comment:
In the `grep` page, we use `search_pattern`, i think we should be consistent :)
```suggestion
`zfgrep {{search_pattern}} {{path/to/file}}`
```

Reply:
Mmm, I could make that change if you really wanted it, but really the whole point of the "F" variant of the grep commands is that the thing you are searching for is taken as a literal string, NOT as a pattern.  So I would prefer to leave it as search_string if up to me...  Thoughts?

Reply:
@navarroaxel Should we use `search_string` or `search_pattern` ?